demos/common/tests/jws_using_mbedtls_ut.c

```diff
@@ -0,0 +1,78 @@
+
+static int initMbedtls( mbedtls_entropy_context * pxEntropyContext,
+                        mbedtls_ctr_drbg_context * pxCtrDrgbContext )
+{
+    int32_t lMbedtlsError = 0;
+
+    /* Set the mutex functions for mbed TLS thread safety. */
+    mbedtls_threading_set_alt( mbedtls_platform_mutex_init,
+                               mbedtls_platform_mutex_free,
+                               mbedtls_platform_mutex_lock,
+                               mbedtls_platform_mutex_unlock );
+
+    /* Initialize contexts for random number generation. */
+    mbedtls_entropy_init( pxEntropyContext );
+    mbedtls_ctr_drbg_init( pxCtrDrgbContext );
+
+    /* Add a strong entropy source. At least one is required. */
```

Comment:
nit - Does the UT really have/need a strong entropy source?

Reply:
Yes for the mbedtls intialization.

Reply:
Do we really need mbedtls?  Could we just mock it out, since ideally a UT shouldn't take any dependencies.

Reply:
I'm not so sure this is a "true" UT. This is to make it easier to verify that this functionality works. So in a way it's more of an e2e test (I can rename if necessary). This could be used in the future if someone were to need to use something like wolfssl and verify that their port worked. This is put in the same area as socket and tls porting code, which aren't tested other than by use in the samples.

Reply:
Ah - makes sense.  I wouldn't name this an _e2e since it's not touching the backend server for anything.

At least in the C SDK I've called something like this an "_int" for integration test.  

You can call it what you want as long as it's not _ut or _e2e :).